README.md

```diff
@@ -2,70 +2,69 @@
-  <a href="https://github.com/open-sauced/pizza/issues">
-    <img src="https://img.shields.io/github/issues/open-sauced/pizza" alt="GitHub issues">
-  </a>
-  <a href="https://github.com/open-sauced/api.opensauced.pizza/releases">
-    <img src="https://img.shields.io/github/v/release/open-sauced/pizza.svg?style=flat" alt="GitHub Release">
-  </a>
-  <a href="https://discord.gg/U2peSNf23P">
-    <img src="https://img.shields.io/discord/714698561081704529.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2" alt="Discord">
-  </a>
-  <a href="https://twitter.com/saucedopen">
-    <img src="https://img.shields.io/twitter/follow/saucedopen?label=Follow&style=social" alt="Twitter">
-  </a>
-</p>
 
+# 📦 Install
+
+#### Homebrew
+
+```sh
+brew install open-sauced/tap/pizza
 ```
-❯ pizza
 
-A command line utility for insights, metrics, and all things OpenSauced
+#### NPM
 
-Usage:
-  pizza <command> <subcommand> [flags]
-
-Available Commands:
-  bake        Use a pizza-oven to source git commits into OpenSauced
-  completion  Generate the autocompletion script for the specified shell
-  help        Help about any command
-  insights    Gather insights about git contributors, repositories, users and pull requests
-  login       Log into the CLI application via GitHub
-  repo-query  Ask questions about a GitHub repository
-  show        Get visual metrics of a repository
-  version     Displays the build version of the CLI
+```sh
+npm i -g pizza
+```
 
-Flags:
-      --beta                Shorthand for using the beta OpenSauced API endpoint ("https://beta.api.opensauced.pizza"). Supersedes the '--endpoint' flag
-      --disable-telemetry   Disable sending telemetry data to OpenSauced
-  -e, --endpoint string     The API endpoint to send requests to (default "https://api.opensauced.pizza")
-  -h, --help                help for pizza
-  -o, --output string       The formatting style for command output (default "table")
+### Docker
 
-Use "pizza [command] --help" for more information about a command.
+```sh
+$ docker run ghcr.io/open-sauced/pizza-cli:v1.1.0
 ```
 
----
+For commands that require access to your file system (like `generate codeowners`), ensure
+you pass a volume to the docker container:
 
-### 📦 Install
+```sh
+$ docker run -v /local/path:/container/path ghcr.io/open-sauced/pizza-cli:v1.1.0 \
+    generate codeowners /container/path
+```
 
-There are several methods for downloading and installing the `pizza` CLI:
+For example, to mount your entire home directory (which may include a `.sauced.yaml` file
+alongside the project you want to generate a `CODEOWNERS` file for):
 
-#### Homebrew
+```sh
+$ docker run -v ~/:/app ghcr.io/open-sauced/pizza-cli:v1.1.0 \
+    codeowners /app/workspace/gopherlogs -c /app/.sauced.yaml
+```
+
+### Go install
+
+Using the Go tool-chain, you can install the binary directly:
 
 ```sh
-brew install open-sauced/tap/pizza
+$ go install github.com/open-sauced/pizza-cli
```

Comment:
I ran this and it failed. It appears to required a version number unless there's something else about the Go ecosystem I'm unaware of.

```
❯ go install github.com/open-sauced/pizza-cli       
go: 'go install' requires a version when current directory is not in a module
        Try 'go install github.com/open-sauced/pizza-cli@latest' to install the latest version

❯ go install github.com/open-sauced/pizza-cli@v1.1.1
go: downloading github.com/open-sauced/pizza-cli v1.1.1

~/dev/work took 3s 
```

Reply:
Whoops! Forgot `@lateset` for the module version.